jsonnet/main.jsonnet

```diff
@@ -420,12 +443,12 @@ local userWorkload =
         },
         namespaces: [$.values.common.namespaceUserWorkload],
         namespaceSelector: $.values.common.userWorkloadMonitoringNamespaceSelector,
-        mixin+: {
-          ruleLabels: $.values.common.ruleLabels,
-          _config+: {
-            prometheusSelector: 'job=~"prometheus-k8s|prometheus-user-workload"',
-          },
-        },
+        //        mixin+: {
```

Comment:
Cant we remove this? Is so, there is another instance of this in lines 464 to 469

Reply:
sure!